File: HelloFrameworks/HelloSubFrameworks/HelloSubFrameworks/ViewController.m
//
//  ViewController.m
//  HelloSubFrameworks
//
//  Created by wesley_chen on 26/03/2018.
//  Copyright © 2018 wesley_chen. All rights reserved.
//

#import "ViewController.h"
//#import <SubFramework1/SubFramework1Manager.h>
#import "SubFramework1Manager.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    SubFramework1Manager *manager1 = [SubFramework1Manager new];
    [manager1 doSomething];
}

@end

File: HelloCoreAnimation/doc/Uberworks-starter/Uberworks/ViewController.swift
//
//  ViewController.swift
//  Uberworks
//
//  Created by Marin Todorov on 10/29/15.
//  Copyright © 2015 Underplot ltd. All rights reserved.
//

import UIKit

func delay(seconds: Double, completion:@escaping ()->()) {
    let popTime = DispatchTime.now() + Double(Int64( Double(NSEC_PER_SEC) * seconds )) / Double(NSEC_PER_SEC)
    
    DispatchQueue.main.asyncAfter(deadline: popTime) {
        completion()
    }
}

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        
    }
}
